### PR TITLE
friendlywelcome: Fix #420

### LINKED ISSUE #420: Missing welcome message: {{Template:Welcome-noteng}}
We have a lot of translated versions of Template:Welcome-noteng, but none in Romanian. Let's add the general one for these cases, please.

### PR DESCRIPTION
**Pull request by ToBeFree**
These changes have NOT been tested. I'm just saving you the editing part of the work, not the testing part. 😉 

Maintainers of this project have write access on my patch-5 branch and are welcome to fix any problems before merging.

closes #420

### CODE REVIEW

File: modules/friendlywelcome.js [hunk 1]
```diff
@@ -594,6 +596,11 @@ Twinkle.welcome.templates = {
 
 	// NON-ENGLISH WELCOMES
 
+	"welcomeen": {
+		description: "welcome for users whose first language is not listed here",
+		linkedArticle: false,
+		syntax: "{{subst:welcomeen}}"
```
Comment (atlight) [inline modules/friendlywelcome.js]: This template should really use Simple English. Starting a sentence with "Efforts" and using complex conjunctive phrases like "in any case" is hardly Simple English.

Comment (ToBeFree) [reply modules/friendlywelcome.js]: Good point, although not an issue with the specific request :) I'll have a look

Comment (ToBeFree) [reply modules/friendlywelcome.js]: Should be fixed now, thanks; I had not thought about that. Feel free to improve further. :)

**Comment by Amorymeltzer** [comment]
`welcomeen` was renamed to `Welcome-noteng` but for consistency's sake, I think it's fine to use the redirect.

**Pull request merged**